source/client/HandView.py

```diff
@@ -2,6 +2,17 @@
 import textwrap
 import client.UIConstants as UIC
 from client.UICardWrapper import UICardWrapper
+import client.ClickableImage as Cli
```

Comment:
Cli is an awkward acronym for  clickable image. I would go CI (that C capital i not C lowercase L, wait now I get why). Maybe ClickImg? Btn is standard and removes vowels and duplicates but Cli seems unclear

Reply:
Also you can import the class itself with "from client.ClickableImage import ClickableImage as ClickImg" and then there is no Cli.ClickableImage in your code.

Reply:
Done